spring-integration-core/src/main/java/org/springframework/integration/aot/CoreRuntimeHints.java

```diff
@@ -103,12 +104,14 @@ public void registerHints(RuntimeHints hints, @Nullable ClassLoader classLoader)
 			reflectionHints.registerType(JsonPathUtils.class, MemberCategory.INVOKE_PUBLIC_METHODS);
 		}
 
+		registerJackson2Hints(reflectionHints);
```

Comment:
I think we need to add `@SuppressWarnings("removal")` to this line or the whole method since we call this deprecated API.